src/helper/ai-generated-exercises/data-conversion.ts

```diff
@@ -303,13 +305,14 @@ function createInteractive(exercise: ExpectedExerciseTypes): Interactive {
 }
 
 function createSolution(
-  exercise: ExpectedExerciseTypes
+  exercise: ExpectedExerciseTypes,
+  licenseId: number
 ): EditorSolutionDocument {
   return {
     plugin: EditorPluginType.Solution,
     state: {
       prerequisite: undefined,
-      licenseId: 1,
+      licenseId,
```

Comment:
sorry did not spot this earlier: 
We can further simplify since solution always defaults to the license of the exercise now, so we don't have to provide it at all.